examples/io/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _io:
+
+Input/Output Examples
+--------------------------
+
+Reading/writing a vareity of radar data using Py-ART.
```

Comment:
vareity to variety

Reply:
> @mgrover1 Was there a reason for removing the getting help section?

Yeah - I think it would be helpful to have a dedicated page.. for now, I removed it since the GitHub button takes you to a similar spot, but open to suggestions. I am thinking we could have a Getting Help Page that summarizes the two main options:
- The email list
- Github Discussions (which would be preferred)

Reply:
That makes sense to me!

Reply:
But yeah - for now, I think it makes sense to remove the link to issues. I can open a separate issues highlighting the that proposed getting help page

Reply:
Changing this now